Backend/src/rest/implementation/UpsertGameRoute.ts

```diff
@@ -24,7 +24,10 @@ export class UpsertGameRoute {
             const updatedGameBoundary: BoundaryGame = this.upsertGameUseCase.upsertGame(gameUpdateBoundary);
             const updatedRestModel: RestGame = this.gameB2RConverter.convert(updatedGameBoundary);
             response.status(200).json(updatedRestModel);
-        } catch (error){
+        } catch (error) {
+            const errorMessage = (error as Error).message;
+            if (errorMessage.includes("ended"))
+                return response.status(403).json((error as Error).message);
```

Comment:
403 reiktų mesti, jeigu useris neturi teisių atlikti veiksmą (pvz, jeigu būčiau adminas, galėčiau atlikti šitą veiksmą), man atrodo šiuo atveju gali mesti kokį 409 (conflict)